chalice/compat.py

```diff
@@ -11,11 +11,16 @@ def pip_import_string():
     # type: () -> str
     import pip
     pip_major_version = pip.__version__.split('.')[0]
+    pip_minor_version = pip.__version__.split('.')[1]
     # Pip moved its internals to an _internal module in version 10.
     # In order to be compatible with version 9 which has it at at the
     # top level we need to figure out the correct import path here.
     if pip_major_version == '9':
         return 'from pip import main'
+    # Pip changed their import structure again in 19.3
+    # https://github.com/pypa/pip/commit/09fd200
+    elif pip_major_version == '19' and int(pip_minor_version) >= 3:
```

Comment:
We can simplify this a bit and just say `(pip_major_version, pip_minor_version) >= (19, 3)` to account for future versions.  You also want to make sure these are ints.